src/Security/EntryPoint/PlainMessageEntryPoint.php

```
@@ -0,0 +1,21 @@
+
+namespace OAT\SimpleRoster\Security\EntryPoint;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Http\EntryPoint\AuthenticationEntryPointInterface;
+
+final class PlainMessageEntryPoint implements AuthenticationEntryPointInterface
+{
+    public function start(Request $request, ?AuthenticationException $authException = null): Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)